client/dashboard/utils/site-plan.ts

```diff
@@ -131,8 +132,8 @@ export function useSitePlanManageURL( site: Site, purchase?: Purchase ) {
 
 	if ( site.plan?.is_free ) {
 		return isCommerceGarden( site )
-			? `${ protocol }//${ host }/setup/woo-hosted-plans?siteSlug=${ site.slug }`
-			: `${ protocol }//${ host }/setup/plan-upgrade?siteSlug=${ site.slug }`;
+			? wpcomLink( `/setup/woo-hosted-plans?siteSlug=${ site.slug }` )
+			: wpcomLink( `/setup/plan-upgrade?siteSlug=${ site.slug }` );
 	}
 
 	if ( isDashboardBackport() ) {
```

Comment:
Maybe it's off topic for this PR, but it looks like this `/purchases/` link is going to break in development so it should use `wpcomLink()` too.

Reply:
I'm referring to line 140

Reply:
Yes, I'll cover the other paths in a follow-up PR!